HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestPageController.java

```diff
@@ -526,14 +522,18 @@ private void processTpmEvents(final List<SupportReferenceManifest> dbSupportRims
 
                             this.referenceDigestValueRepository.save(newRdv);
                         }
-                    } catch (CertificateException | NoSuchAlgorithmException | IOException e) {
+                    } catch (CertificateException e) {
```

Comment:
Since we are printing the same stack trace for all three exceptions, why did you separate them out into their own catch statement? Do you believe that we will add more code to each exception case?

Reply:
This change was automatically applied by checkstyles.  There are instances in the tcg_rim_tool of exceptions broken out into distinct catch blocks for specific handling, which can be applied here if appropriate.